packages/manager/apps/telecom/src/components/telecom/telephony/group/number/feature/ovhPabx/dialplan/extension/rule/edit/edit.controller.js

```diff
@@ -208,6 +210,78 @@ export default class DialplanExtensionRuleEditCtrl {
   onRuleActionChange() {
     this.parentCtrl.popoverStatus.move = false;
     this.rule.actionParam = '';
+    const step = this.parentCtrl.extension.position;
+
+    switch (this.rule.action) {
+      case 'bridge':
```

Comment:
This saddens me just for `atInternet`

Reply:
yes the tracking name choice doesn't make it easy to implement cleanly :/